spec/helpers/persons_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the PersonsHelper. For example:
+#
+# describe PersonsHelper do
```

Comment:
commented code and pending folder, is this valid?

Reply:
no, It isn't.